resolwe/flow/executors/prepare.py

```diff
@@ -26,17 +26,22 @@ def extend_settings(self, data_id, files):
             filenames, values are the objects that will be serialized
             into those files. Standard filenames are listed in
             :class:`resolwe.flow.managers.protocol.ExecutorFiles`.
+        :param extra: Extra raw files dictionary describing additional raw
+            file content that should be created. Keys are filenames, values
+            are the strings that should be written into those files.
         """
+        data = Data.objects.get(pk=data_id)
```

Comment:
As we are getting just a single object this doesn't make any difference in performance or the number of queries as even prefetching requires an additional query. We could use `select_related` here instead of prefetch to combine everything into a single join.